cmake/ctest/drivers/geminga/cron_driver.sh

```diff
@@ -35,17 +35,24 @@ export TDD_HTTPS_PROXY=$https_proxy
 
 . ~/.bashrc
 
+# Machine independent cron_driver:
+SCRIPT_DIR=`cd "\`dirname \"$0\"\`";pwd`
+
+
+export export MODULEPATH=$SCRIPT_DIR:$MODULEPATH
+
 # If you update the list of modules, go to ~/code/trilinos-test/trilinos/ and
 # do "git pull". Otherwise, the tests could fail on the first night, as we
 # would first run old cron_driver.sh and only then pull
 
 # ===========================================================================
 export CTEST_CONFIGURATION="default"
-module load sems-cmake/3.17.1
-module load sems-gcc/5.3.0
-module load sems-openmpi/1.10.1
-module load sems-superlu/4.3/base
-module load sems-git/2.10.1
+# module load sems-cmake/3.17.1
+# module load sems-gcc/5.3.0
+# module load sems-openmpi/1.10.1
+# module load sems-superlu/4.3/base
+# module load sems-git/2.10.1
+module load muelu-gcc
```

Comment:
since when was there a muelu-gcc module? :open_mouth:

Reply:
since I added it in this PR ;-)